app/controllers/SignUpController.scala

```diff
@@ -58,6 +58,7 @@ class SignUpController @Inject() (
             lastName = Some(data.lastName),
             fullName = Some(data.firstName + " " + data.lastName),
             email = Some(data.email),
+            role = Some("Student"),
```

Comment:
IMHO "Student" and other role-Strings should be extracted into app/utils/silhouette/Authorization.scala (something like ```val ROLE_STUDENT = "Student"```) and then be referenced instead of a direct String. This avoids typos etc.